components/AudioPlayer.js

```diff
@@ -0,0 +1,134 @@
+  const [duration, setDuration] = useState(0);
+  const [currentTime, setCurrentTime] = useState(0);
+
+  // references
+  const audioPlayer = useRef();   // reference our audio component
+  const progressBar = useRef();   // reference our progress bar
+  const animationRef = useRef();  // reference the animation
+
+  useEffect(() => {
+    const seconds = Math.floor(audioPlayer.current.duration);
+    setDuration(seconds);
+    progressBar.current.max = seconds;
+  }, [audioPlayer?.current?.loadedmetadata, audioPlayer?.current?.readyState]);
+
+  const calculateTime = (secs) => {
+    const minutes = Math.floor(secs / 60);
+    const returnedMinutes = minutes < 10 ? `0${minutes}` : `${minutes}`;
+    const seconds = Math.floor(secs % 60);
+    const returnedSeconds = seconds < 10 ? `0${seconds}` : `${seconds}`;
+    return `${returnedMinutes}:${returnedSeconds}`;
+  }
+
+  const togglePlayPause = () => {
+    const prevValue = isPlaying;
+    setIsPlaying(!prevValue);
+    if (!prevValue) {
+      audioPlayer.current.play();
+      animationRef.current = requestAnimationFrame(whilePlaying)
+    } else {
+      audioPlayer.current.pause();
+      cancelAnimationFrame(animationRef.current);
+    }
+  }
+
+  const whilePlaying = () => {
+    progressBar.current.value = audioPlayer.current.currentTime;
+    changePlayerCurrentTime();
+    animationRef.current = requestAnimationFrame(whilePlaying);
+  }
+
+  const changeRange = () => {
+    audioPlayer.current.currentTime = progressBar.current.value;
+    changePlayerCurrentTime();
+  }
+
+  const changePlayerCurrentTime = () => {
+    progressBar.current.style.setProperty('--seek-before-width', `${progressBar.current.value / duration * 100}%`)
+    setCurrentTime(progressBar.current.value);
+  }
+
+  const backThirty = () => {
+    progressBar.current.value = Number(progressBar.current.value - 30);
+    changeRange();
+  }
+
+  const forwardThirty = () => {
+    progressBar.current.value = Number(progressBar.current.value + 30);
+    changeRange();
+  }
+
+  return (
+    <section
+      className={`${utilsStyles.bg_white} ${styles.audio_player_section}`}
+      id="episodes"
+    >
+      <div className={styles.audio_player_section_container}>
+        <h2 className={utilsStyles.font_black}>
+          <span className={utilsStyles.font_accent}>E</span>pisodes
+        </h2>
+        <div className={styles.image_wrapper}>
+          <Image
+            priority
+            src="/assets/sample_profile_pic.jpeg"
+            layout="fill"
+            objectFit="cover"
+            objectPosition="left top"
+            alt="profile-dummy"
+            style={{boxShadow: "0 0 10 black"}}
+          />
+        </div>
+        {/* episode_info_section */}
+        <div className={styles.episode_info_section}>
+        <div className={styles.episode_title}>
+          <h3>003 Occupy</h3>
+          <h4>-Raid of Zuccati Park-</h4>
+          <h5>Nov.15th, 12-2am On Air</h5>
```

Comment:
```suggestion
          <p>Nov.15th, 12-2am On Air</p>
```
Probably this is not a title and more like text so I suggest that you replace `h5` with `p`.